compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -111,6 +112,17 @@ internal class ComposeLayer {
             if (a11yDisabled) return null
             val controller =
                 scene.mainOwner?.accessibilityController as? AccessibilityControllerImpl
+            controller?.parent = component.parent
+            val resetFocusTimer = Timer(100) {
+                // Listener spawns asynchronous notification post procedure, reading current focus owner
+                // and its accessibility context. This timer is used to deal with concurrency
+                // TODO Find more reliable procedure, sometimes updates are missing
+                _component.canvas.requestFocus(FocusEvent.Cause.MOUSE_EVENT)
```

Comment:
Hmm, not sure if I understand the logic. Why do we forcefully steal focus, why don't we do at least
```
SwingUtilities.invokeLater {
   _component.canvas.requestFocus(FocusEvent.Cause.MOUSE_EVENT)
}
```
or request it on some component init.

Reply:
`SwingUtilities.invokeLater` does not work. Seems that focus is released before notification callback start